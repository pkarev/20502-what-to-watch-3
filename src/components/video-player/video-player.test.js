import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const movie = {
  trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  poster: `img/the-grand-budapest-hotel-poster.jpg`,
};

it(`Render VideoPlayer`, () => {
  const tree = renderer
  .create(
      <VideoPlayer
        src={`${movie.trailer}`}
        poster={`${movie.poster}`}
      />, {
        createNodeMock: () => ({})
      })
  .toJSON();

  expect(tree).toMatchSnapshot();
});
