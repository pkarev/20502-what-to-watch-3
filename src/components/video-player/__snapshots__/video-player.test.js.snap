// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer 1`] = `
<video
  controls={false}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  poster="img/the-grand-budapest-hotel-poster.jpg"
  style={
    Object {
      "objectFit": "cover",
      "width": "100%",
    }
  }
>
  <source />
</video>
`;
